### PR TITLE
DC-485 Clean up some additional code from imported flows parser classes

### PR DESCRIPTION
**Pull request by FrancescoBuc86**
- Remove some useless methods/classes

### CODE REVIEW

**Comment by FrancescoBuc86** [comment]
https://opennms.atlassian.net/browse/DC-485

File: minion/flows/parser/src/main/java/parser/transport/IpFixMessageBuilder.java [hunk 1]
```diff
@@ -592,40 +553,33 @@ public FlowMessage.Builder buildMessage(final Iterable<Value<?>> values, final R
...
+                    direction = Direction.EGRESS;
+                    break;
+            }
+        }
+        return direction;
     }
 
-    public void setFlowSamplingIntervalFallback(final Long flowSamplingIntervalFallback) {
-        this.flowSamplingIntervalFallback = flowSamplingIntervalFallback;
+    static void buildDeltaSwitsched(FlowMessage.Builder builder, Long flowActiveTimeout, Long flowInactiveTimeout) {
```
Comment (patrick-schweizer) [inline minion/flows/parser/src/main/java/parser/transport/IpFixMessageBuilder.java:576]: typo? Swits**c**hed

**Review by patrick-schweizer — COMMENTED** [review]
I found one typo. Maybe we can eliminate the code smells before merging?

**Review by patrick-schweizer — APPROVED** [review]
:+1: 
Coverage will be addressed in https://opennms.atlassian.net/browse/DC-489

**Review by freddy61025 — COMMENTED** [review]
how about the packages prefix? Will it cover in this task?

e.g. package listeners.utils; > package org.opennms.minion.flows.listeners.utils;

**Pull request merged**